apps/staff/app/print-request/[id]/page.tsx

```
@@ -148,6 +149,30 @@ export default function PrintRequestPage() {
         </div>
 
+        {/* Application Responses (formData) */}
+        {request.formData && Object.keys(request.formData).length > 0 && (
+          <div className="mb-6">
+            <h2 className="text-xl font-semibold text-gray-900 mb-3 border-b border-gray-300 pb-2">
+              {REQUEST_TYPE_LABELS[request.type] || request.type.replace(/_/g, ' ')} — Application
+              Responses
+            </h2>
+            <div className="space-y-3">
+              {getFormDataDisplayItems(request.type, request.formData).map((item, index) => (
+                <div key={index} className="break-inside-avoid">
```

Biome (2.4.10)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)